### DISCUSSION TITLE
Can Multiple SkyCrypt Instances use the same API Key?

### DISCUSSION THREAD

**Discussion by WarpWing** [HypixelDev/PublicAPI#474] [Q&A] [answered]
Hey all! Quick question to the devs.

I'm working on a infrastructure rework of SkyCrypt that calls using Kubernetes in order to provide SkyCrypt with a more cluster based solution. If I feed all of the SkyCrypt Pods the same API Key, is that considered an abuse of a API Key or no. These pods will all Round Robin Loadbalance the traffic from a Traefik router and will point to one domain(obviously), not multiple. Just curious if this is a viable approach. This is not an offical project YET but this is simply just for testing purposes. Anyways I just want to make that I'm not abusing the API. Thanks!

**Comment by ConnorLinfoot** [comment, marked as answer]
There is nothing that should stop you from what is effectively using a single key on multiple services as you are still of course limited to the restrictions on said key.

**Reply by WarpWing** [reply]
Thanks, Conner. Just want to make sure I'm not abusing the API. :)
